src/new/connection.ts

```diff
@@ -56,7 +58,9 @@ class WalletConnection extends BaseConnection {
 
   streamController: StreamController | null = null;
 
-  constructor(options: ConnectionParams) {
+  streamWindowSize: number | undefined;
```

Comment:
Should it be in the `streamController` object?

Reply:
We could move this to the stream manager, but I think that this is a Connection level configuration.
What do you think?

Reply:
Got it. Ok. As discussed, I guess it might be worth refactoring it later. Thanks!